src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -1750,7 +1750,7 @@ bool ISingleCanSaveOnnx.SaveAsOnnx(OnnxContext ctx, string[] scoreProbablityColu
             var node = ctx.CreateNode(opType, new[] { scoreProbablityColumnNames[0], slopVar }, new[] { mulNodeOutput }, ctx.GetNodeName(opType), "");
 
             opType = "Add";
-            var betaVar = ctx.AddInitializer(-0.0000001f, "Slope");
+            var betaVar = ctx.AddInitializer((float)(-Offset), "Slope");
             var linearOutput = ctx.AddIntermediateVariable(null, "linearOutput", true);
```

Comment:
So I've updated this line in my last PR #4699 which just has been merged. You should sync this to the master so you get the updated version. Thanks!